ui/tests/integration/components/oidc/provider-form-test.js

```diff
@@ -84,7 +84,7 @@ module('Integration | Component | oidc/provider-form', function (hooks) {
       .dom('[data-test-component="search-select"]#allowedClientIds')
       .exists('Limited radio button shows clients search select');
     await click('[data-test-component="search-select"]#allowedClientIds .ember-basic-dropdown-trigger');
-    assert.dom('li.ember-power-select-option').hasText('some-app', 'dropdown renders clients');
+    assert.dom('li.ember-power-select-option').hasTextContaining('some-app', 'dropdown renders client name');
```

Comment:
the `client_id` is generated by the backend and will change every test run, so can't confirm the exact id text